src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,34 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    public static final int ARRAY_LENGTH = 12;
+    public static final int EMPTY = -1;
+    private final int[] numbers = new int[ARRAY_LENGTH];
```

Comment:
maybe something like `MAX_ARRAY_LENGTH` so  it won't be confused with `total` variable?
```suggestion
    public static final int MAX_ARRAY_LENGTH = 12;
    public static final int EMPTY = -1;
    private final int[] numbers = new int[MAX_ARRAY_LENGTH];
```